src/components/Votes.jsx

```diff
@@ -0,0 +1,22 @@
+import useVote from "./useVote";
+
+const Votes = ({ article_id, votes }) => {
+  const { votedStatus, handleVote, isSubmitting } = useVote(article_id);
+
+  return (
+    <button
+      onClick={handleVote}
+      className="likeButton"
+      disabled={isSubmitting}
+    >
+      {votedStatus ? (
+        <span className="likedEmoji">{votes + 1} 👎 </span>
```

Comment:
Your up/down vote functionality is a bit confusing - Ideally rather than having the thumbs up turn to a thumbs down and the number increase, I would ideally have a up and down button both always showing and based on the one clicked, update the votes with + or - accordingly.